dev-python/pymdown-extensions/pymdown-extensions-8.1.1.ebuild

```diff
@@ -38,6 +38,20 @@ BDEPEND="
 distutils_enable_tests pytest
 
 python_prepare_all() {
+	# Fails on whitspaces
```

Comment:
That doesn't fix tests. That just stupidly skips them without actually spending 3 minutes to think what's wrong.